MyfansDownloader_unified.py

```
@@ -0,0 +1,125 @@
+        
+        port = 5000
+        url = f"http://localhost:{port}"
+        
+        print(f"Starting web server on {url}")
+        print("Opening browser in 2 seconds...")
+        
+        # Open browser after a short delay
+        time.sleep(2)
+        try:
+            webbrowser.open(url)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)